src/modules/layouts/hooks/use-grouping.ts

```
@@ -1,14 +1,163 @@
+  const onChangeOption = useGroupingStore((state) => state.onSetGroupingValue);
+  const onChangeSort = useGroupingStore((state) => state.onSetGroupingSort);
+
+  const isGrouping = grouping !== null;
+  const groupingType = grouping?.columnDef.meta?.variant; 
+
+  const getGroupingDescription = () => {
+    if (!groupingValue || !groupingType) return "";
+    
+    switch (groupingType) {
+      case "numeric":
+        const numericValue = groupingValue as NumericBy;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)